app/views/jobseekers/alert_mailer/alert.text.erb

```diff
@@ -1,15 +1,16 @@
 # <%= t(".summary.#{subscription.frequency}", count: @vacancies.count) %>
 
 ---
-
-<%- @vacancies.each do |vacancy| %>
+<%- @vacancies.each_with_index do |vacancy, index| %>
```

Comment:
It doesn't look like we are using the `index`. Is this line change needed?

Reply:
oops nope, just using it for testing a scenario but forgot to remove this line 😳